workers/notifier/build.gradle.kts

```diff
@@ -34,6 +34,12 @@ plugins {
 
     // Apply third-party plugins.
     alias(libs.plugins.jib)
+    alias(libs.plugins.jakartaMigration)
+}
+
+jakartaeeMigration {
+    includeTransform("com.atlassian.jira:jira-rest-java-client-core")
+    configurations.forEach(::transform)
```

Comment:
To align with ORT's code, I'd prefer to keep the filtering here as well, even if it wouldn't be strictly required.